packages/components/src/community/sidebar-community/sidebar-community.tsx

```diff
@@ -18,21 +21,24 @@ type Props = {
     membersCount: number
     imageUrl: string
```

Comment:
imageSrc or just src to be consistent with avatar images?

Reply:
done in https://github.com/status-im/status-web/pull/370/commits/f5aaf7edc2dfabb448b9bbe832f6441d00bb82ca